docs/0.3-DRAFT/index.md

```diff
@@ -346,11 +383,108 @@ start with `https://w3id.org/ro/crate/`.
+
+ {
+   "@id": "_:localid:my-repo:my-id",
+   "@type": "PropertyValue",
+   "name": "my-repo",
+   "value": "my-id"
+ }
+
+  {
+   "@id": "_:localid:other-repo:https://doi.org/10.4225/59/59672c09f4a4b",
+   "@type": "PropertyValue",
+   "name": "other-repo",
+   "value": "https://doi.org/10.4225/59/59672c09f4a4b"
+ }
+ ```
+
+The _RO-Crate JSON-LD_ MUST contain an
+_RO-Crate Metadata File Descriptor_ of type [CreativeWork] which describes the `Root Data Entity`, with an [about]
+property referencing the _Root Data Entity_, which MUST have an `@id` of `./`.
+
+```json
+{
+    "@type": "CreativeWork",
+    "@id": "ro-crate-metadata.jsonld",
+    "identifier": "ro-crate-metadata.jsonld",
+    "about": {"@id": "./"}
+}
+
+```
+
+The _RO-Crate Metadata File Descriptor_ MAY contain information such as
+licensing for the _RO-Crate Metadata File_ so metadata can be licensed
+separately from Data.
+ 
+
+#### Other _Data Entity_ Identifiers
+
+Other data entities (`File`s and `Dataset`s) MUST have IDs which are paths relative to the root. 
+
+For Datasets which correspond to a directory the the `@id` must end in `/` and for DataEntities of @type `File` the `@id` must be a path - which SHOULD NOT begin with `./`, for example:
+
+```json
+{
+      "@id": "pics/",
+      "@type": "Dataset",
+      "description": "This directory contains the images for the research project",
+      "hasPart": [
+        {
+          "@id": "pics/2017-06-11 12.56.14.jpg"
+        },
+        {
+          "@id": "pics/sepia_fence.jpg"
+        },
+        {
+          "@id": "pics/thumbs/"
+        }
+      ],
+      "identifier": "pics/",
+      "name": "Pictures"
+    },
+    {
+      "@id": "pics/2017-06-11 12.56.14.jpg",
+      "@type": "File ImageObject",
+      "contentSize": "5114778",
+      "creator": {
+        "@id": "http://orcid.org/0000-0002-3545-944X"
+      }
+
+```
+
+Note that in this example the `File` has an array of two values for `@id`. It
+MUST have the `@type` file, other types MAY be used as well.
+
+#### _Contextual Entity_ identifiers
+
+If possible, _Contextual Identifiers_ SHOULD have URIs as values for their `@id`:
+*   For People participating in the research process: [ORCID](https://orcid.org) identifiers.
+*   For [Organization](http://schema.org/Organization)s including [funder](http://schema.org/funder)s,  Research Organization Registry URIs. [https://ror.org/](https://ror.org/)
+*   For items of type [Place], a geonames URL.
+*   For file formats, a [Pronom] URL, for example <https://www.nationalarchives.gov.uk/PRONOM/fmt/831>.
+*   In other cases, a stable, useful URL that resolves to something helpful to describe the entity.
+
+If there is no usable URI to identify an entity, _Contextual Entities_ MAY have local IDs (prefaced with a `#` or JOSN blank node identifiers prefaced with `_:`.
+
+Identifiers for _Contextual Entities_ MUST NOT use `@id`s which can be construed as paths. So `#NikonCamera1` is acceptable but `NikonCamera1` is NOT.
```

Comment:
I think lower this to SHOULD - it would be valid - particularly on the web, that the contextual entity has a direct resource representation (which is not a `File`).  You have earlier mentioned instruments with wiki pages, I don't see why you could not do that here as well, but you would need to know what you are doing (hence thoughtfully violate a SHOULD)